src/igvfd/schemas/sample.json

```diff
@@ -90,6 +90,13 @@
             "comment": "Date should be submitted in as YYYY-MM-DD.",
             "type": "string",
             "format": "date"
+        },
+        "sorted_fraction": {
+            "title": "Sorted Fraction",
+            "description": "Links to a larger sample from which this sample was obtained through sorting.",
+            "comment": "The related sample must have the same properties or this will result in an audit.",
```

Comment:
please change the comment into: "The parent sample is expected to have similar properties to it's fraction."

Reply:
Done